typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -1391,6 +1391,21 @@ class E:
                     x = 1
                 self.assertIsInstance(E(), D)
 
+        def test_collections_protocols_allowed(self):
+            @runtime
```

Comment:
`@runtime_checkable`?

Reply:
Well, they both work in `typing_extensions` (there is the alias), but it makes sense to use the canonical one.